sdcm/scan_operation_thread.py

```diff
@@ -307,28 +296,81 @@ def execute_query(self, session, cmd: str):
         session.default_consistency_level = ConsistencyLevel.ONE
         return session.execute_async(cmd)
 
-    def run_scan_operation(self, scan_operation_event, cmd: str = None):  # pylint: disable=too-many-locals
+    def reset_output_files(self):
+        self.normal_query_output.close()
+        self.reversed_query_output.close()
+        self.reversed_query_output = tempfile.NamedTemporaryFile(mode='w+', delete=False,  # pylint: disable=consider-using-with
+                                                                 encoding='utf-8')
+        self.normal_query_output = tempfile.NamedTemporaryFile(mode='w+', delete=False,  # pylint: disable=consider-using-with
+                                                               encoding='utf-8')
+
+    def _run_shell_command(self, cmd: str, verbose: bool = True) -> (str, str):
```

Comment:
Why not using LocalRemoter ? i.e SCT has this kind of helpers already

Reply:
fixed.